src/signs/ui/search/MesZLContent.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from 'react'
+import './Signs.css'
+import ExternalLink from 'common/ExternalLink'
+import unified from 'unified'
+import remarkParse from 'remark-parse'
+import remark2rehype from 'remark-rehype'
+import raw from 'rehype-raw'
+import stringify from 'rehype-stringify'
+
+export default function MesZlContent({
+  mesZl,
+}: {
+  mesZl: string
+}): JSX.Element | null {
+  const [mesZlHead, setMesZlHead] = useState<string | null>(null)
+  const [mesZlBody, setMesZlBody] = useState<string | null>(null)
+
+  const mesZlLines = mesZl.split('\n')
+
+  const mesZlFormatted = mesZlLines
+    .slice(1, 6)
```

Comment:
@jlaasonen Enrique decided just to show 6 Lines for now until the expanded Sign Display is finished. Otherwise it would be too many lines for the popover. (We cut at Line 6 and display a ... for now).